lib/task_listener.js

```diff
@@ -229,7 +228,21 @@ export default class TaskListener extends EventEmitter {
     }
   }
 
-  addRunningTask(runningState) {
+  async recordCapacity () {
+    if(this.lastKnownCapacity === undefined) {
+      await this.availableCapacity();
+    }
+    this.runtime.stats.record('capacityOverTime', {
+      duration: Date.now() - this.lastTaskEvent,
+      availableCapacity: this.lastKnownCapacity,
+      runningTasks: this.runningTasks.length
```

Comment:
Can we add one extra field called `totalCapacity: this.lastKnownCapacity + this.runningTasks.length`.
Because when `totalCapacity` is zero, we know the worker hasn't started yet... Or is waiting for a resource (say a device) before it can run anything.

Also it's nice to be able to say, what is average of `runningTasks` for all with `totalCapacity = 16` (ie big machines).
This would tell us if we utilize them. I know we can calculate this, but I suspect storing it would make it faster? Or, hmm maybe not... because it's not a tag... maybe we should add it as a string value so we can make a 0.9 tag.